scripts/rebase.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -ex
+
+base_brach=${1:-master}
+head_branch=${2:-next}
+
+git fetch origin "$base_brach"
+git fetch origin "$head_branch"
+git checkout -b "$head_branch" "origin/$head_branch"
+
+is_rebasing=1
+
+exitcode=0 && git rebase "origin/$base_brach" || exitcode=$?
```

Comment:
why chaining the setting of `exitcode` to the rebasing & why setting it at all, and not just using `$?`?

Reply:
Because I want to keep the exit code and if I were just using `$?` it would be the exit code of the last command, which might not be what I'm interested in.